### PR TITLE
Moving to metadata.json

### PR DESCRIPTION
**Pull request by nibalizer**

### CODE REVIEW

**Comment by adrienthebo** [comment]
@nibalizer thanks for doing this!

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "puppetcommunity-alternatives",
+  "version": "0.3.0",
+  "author": "Adrien Thebo <adrien@somethingsinistral.net",
+  "summary": "Maintain symbolic links determining default commands",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppet-community/puppet-alternatives",
+  "project_page": "https://github.com/puppet-community/puppet-alternatives",
+  "issues_url": "https://github.com/puppet-community/puppet-alternatives",
+  "operatingsystem_support": [
```
Comment (adrienthebo) [inline metadata.json]: @jhoblitt @jsosic we support redhat as well, right?

**Comment by nibalizer** [comment]
@adrienthebo do you want to push this to forge.puppetlabs.com/puppet or keep pushing it to adrienthebo?

**Comment by adrienthebo** [comment]
The last update I pushed was to my namespace because of time constraints and lack of access to someone push publishing access; I'm totally fine with this moving under the community namespace.

**Comment by nibalizer** [comment]
@adrienthebo I have push access as does @igalic, ping whenever you want a release :)

**Comment by igalic** [comment]
or, we establish a new way of pushing that's not based on gpg, because that's been pronounced dead :P

**Comment by raphink** [comment]
How about using Travis CI deploy with encrypted credentials? We do this for
camptocamp's modules so tags on GitHub automatically upload versions to the
forge.

**Comment by igalic** [comment]
i'm cool with that too… whatever gets us *easy* releases

**Comment by nibalizer** [comment]
I have a hard time believing that travis holding a password is secure. That being said a lot of people do it. If people think its safe and the correct way to go then I'm down.

**Comment by jhoblitt** [comment]
It's either put the credentials in as a travis secret or standard up an external service that receives a webhook.  The former sounds like less of a hassle to me.

**Comment by raphink** [comment]
Each Travis repository has a pair a keys associated with it, that you can
use to encrypt values using the travis command line tool. The values can
only be decrypted when running a build on the given repository. I think
that's secure enough for this kind of need.

**Comment by raphink** [comment]
Here is an example we use in production:

https://github.com/camptocamp/puppet-openldap/blob/master/.travis.yml

See the deploy section, where the password is hashed. I can't decrypt the
value, it can only be accessed in the build triggered from our repository
(not from forks or PRs either).

**Comment by raphink** [comment]
And the result of the Travis build on a tag
https://travis-ci.org/camptocamp/puppet-openldap/jobs/51198716#L780

**Comment by igalic** [comment]
@nibalizer: make it so, number one.

**Comment by nibalizer** [comment]
Okay, added redhat support to metadata.json. As for deployment we have some docs here https://puppet-community.github.io/docs.html#publishing

**Pull request merged**